abdiff/webapp/app.py

```diff
@@ -89,63 +89,49 @@ def run(run_timestamp: str) -> str:
         except FileNotFoundError:
             transform_logs = "'logs.txt' not found for transform logs"
 
```

Comment:
Something that caught my attention when reviewing the web app is this: 
![image](https://github.com/user-attachments/assets/5b6f593b-091d-4472-a38a-584c301e147c)

Will this only show the logs if `PRESERVE_ARTIFACTS="true"`? Can we move `logs.txt` to the `run/YYYY-MM-DD_HH-MM-SS/` directory, such that `logs.txt` is preserved even after `transformed/` directory is removed? 

The[ example-job-3 test fixture](https://github.com/MITLibraries/transmogrifier-ab-diff/tree/TIMX-385-records-preview/tests/fixtures/jobs/example-job-3/runs/2024-11-07_03-56-22) seems to suggest that this is what _should_ be happening. 🤔

Reply:
100% agreed.  I _almost_ put this into the PR, but wanted to keep it tightly scoped to the records table.

What I should have done is acknowledge the issue, but pointed to these two open tickets:
- https://mitlibraries.atlassian.net/browse/TIMX-368
- https://mitlibraries.atlassian.net/browse/TIMX-382

As with many things in this project, would hate to apply a band-aid to what feels like needs a little updating and rethinking (given scale).  Are you okay @jonavellecuerdo just keeping as-is for the moment, and relying on these tickets to resolve?

Reply:
Agree with that approach as well